python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -423,55 +573,12 @@ def runDataRef(self, sensorRef, templateIdList=None):
                         preConvPsf = srcPsf
                     afwMath.convolve(destMI, srcMI, preConvPsf.getLocalKernel(), convControl)
                     exposure.setMaskedImage(destMI)
-                    scienceSigmaPost = scienceSigmaOrig*math.sqrt(2)
-                else:
-                    scienceSigmaPost = scienceSigmaOrig
 
                 # If requested, find sources in the image
                 if self.config.doSelectSources:
```

Comment:
Ideally this `if self.config.doSelectSources:` code block would get moved into its own method. This would include the code from line 408 above where I made a similar comment. This might be beyond the scope of this ticket, though.